iree/compiler/Dialect/VM/Conversion/VMToEmitC/ConvertVMToEmitC.cpp

```diff
@@ -28,6 +30,46 @@ namespace iree_compiler {
 
 namespace {
 
+LogicalResult sanitizeSymbolNames(IREE::VM::ModuleOp &moduleOp) {
+  auto replaceIllegalChars = [](std::string &symbolName) {
+    std::regex illegalCharactersRe("<|>");
+    auto beginIt = std::sregex_iterator(symbolName.begin(), symbolName.end(),
+                                        illegalCharactersRe);
+    auto endIt = std::sregex_iterator();
+
+    // Replace illegal characters in symbol name with "_" using iterator
```

Comment:
style: proper punctuation on comments (meaning a period at the end of all these); see https://google.github.io/styleguide/cppguide.html#Punctuation,_Spelling_and_Grammar
```suggestion
    // Replace illegal characters in symbol name with "_" using iterator.
```

(here and below)

Reply:
Good catch - thanks! I'll change it!